index.js

```diff
@@ -70,6 +70,38 @@ Builder.prototype.build = function(moduleName, outFile, opts) {
   });
 };
 
+var operatorFunctions = {
+  '+': builder.addTrees,
+  '-': builder.subtractTrees
+};
+
+Builder.buildExpression = function(firstModule, operations, cfg) {
```

Comment:
Sure, if they're all binary then no reason to allow arrows - just more support for malformed input down the road.